File: Restaurant-information-system/src/main/java/com/galaxy/Restaurantinformationsystem/DTO/UserRegisterFormDto.java
package com.galaxy.Restaurantinformationsystem.DTO;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserRegisterFormDto {
    @Schema(description = "이메일", example = "user@user.com")
    private String email;
    @Schema(description = "비밀번호", example = "1234")
    private String password;
    @Schema(description = "이름", example = "user")
    private String name;
}

File: src/main/java/com/galaxy/restaurant/information/system/dto/UserRegisterFormDto.kt
package com.galaxy.restaurant.information.system.dto

import io.swagger.v3.oas.annotations.media.Schema

@Schema(name = "UserRegisterFormDto", description = "유저 등록 폼 DTO")
data class UserRegisterFormDto(
    @field:Schema(name = "email", description = "이메일", example = "eamil@exmaple.com")
    var email: String,
    @field:Schema(name = "password", description = "비밀번호", example = "password")
    var password: String,
    @field:Schema(name = "name", description = "이름", example = "name")
    var name: String
) {
    override fun toString(): String {
        return "UserRegisterFormDto(email='$email', password='$password', name='$name')"
    }
}
